pom.xml

```diff
@@ -181,7 +181,7 @@
     </build>
 
     <properties>
-        <cas.version>4.1.0-SNAPSHOT</cas.version>
+        <cas.version>4.1.0</cas.version>
```

Comment:
no per release policy patch releases never have API changes, but there is a chance you may run into dependency issues which is also unlikely but possible.